src/main/java/htsjdk/tribble/gff/Gff3Codec.java

```diff
@@ -220,19 +227,22 @@ private void validateFeature(final Gff3Feature feature) {
 
     @Override
     public Feature decodeLoc(LineIterator lineIterator) {
+        return decodeShallow(lineIterator);
```

Comment:
This is probably more expensive than the previous decodeLoc implementation, but I think it's a pretty underused operation anyway so it is unlikely to make a difference anywhere.  I think it's only used in certain reindexing operations.